sites/public/pages/eligibility/age.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+Age
+Prompts the user for their age to filter for properties that are age dependent.
+*/
+import {
+  AppearanceStyleType,
+  Button,
+  FormCard,
+  t,
+  Form,
+  Field,
+} from "@bloom-housing/ui-components"
+import FormsLayout from "../../layouts/forms"
+import { useForm } from "react-hook-form"
+import { useFormConductor } from "../../lib/hooks"
+import "./age.scss"
+import React from "react"
+
+const EligibilityAge = () => {
+  // The conductor is not yet implemented.
```

Comment:
I was getting a little tripped up by trying to understand the "There is no step defined which matches age" server logs I was seeing, but this clears it up! A nit: is it easy to change this comment (and the others like it in the other files) to briefly say what someone would have to do to implement the conductor piece?

Reply:
Going to work on this next, so hopefully the comment won't be here for long!